docs/support/tutorials/gis/eo_guide.md

```diff
@@ -0,0 +1,255 @@
+# Earth Observation guide
+
+This guide aims to help users who wish to work with Earth Observation (EO) data using CSC's computing resources. The purpose of this guide is to help you find the right data and tools for your raster data based EO tasks. If you are interested in the funamentals of EO, please check the resources at the bottom of the page first.
+
+**What are the benefits of using EO data**
+
+* Possibility to observe wide area at same time
+* Non-intrusive
+* Same sensor for different parts of the world
+* Time series 
+
+**EO data in raster format**
+
+* One file per band or multiband files
+* Grid of pixel values
+* Example of continuous data
+* Georeference: coordinate for the top left pixel , the size of each pixel in the X direction, the size of each pixel in the Y direction, and the amount (if any) by which the product is rotated.
+
+
+## What data do I need?
+
+What to consider when chosing data:
+
+* Sensor
+    * Optical, Multispectal: [Landsat](https://landsat.gsfc.nasa.gov/), [MODIS](https://modis.gsfc.nasa.gov/), [Copernicus Sentinel-2](https://sentinel.esa.int/web/sentinel/missions/sentinel-2),[Pleiades](https://pleiades.cnes.fr/en/PLEIADES/index.htm), [Spot](https://www.intelligence-airbusds.com/imagery/constellation/spot/), [Planet](https://www.planet.com/products/planet-imagery/), [WorldView](http://worldview3.digitalglobe.com/), [Proba-V](https://earth.esa.int/eogateway/catalog/proba-v-1km-333m-and-100m-products), ...
+    * RADAR (Radio Detection and Ranging) -> SAR (Synthetic Aperture Radar): [Copernicus Sentinel 1](https://sentinel.esa.int/web/sentinel/missions/sentinel-1), [Radarsat](https://www.asc-csa.gc.ca/eng/satellites/radarsat/), [TanDEM-X](https://www.dlr.de/content/en/missions/tandem-x.html), [TerraSAR-X](https://www.dlr.de/content/en/missions/terrasar-x.html), [ICEYE](https://www.iceye.com/),...
+    * LiDAR (Light Detection and Ranging): [ICESat](https://icesat-2.gsfc.nasa.gov/), [GEDI](https://gedi.umd.edu/), ...
+    * Database of all current and future EO missions and instrument information can be found in the [CEOS EO handbook database](http://database.eohandbook.com/database/instrumenttable.aspx)
```

Comment:
Move this line to the end (or beginning of this chapter). The search there has other options also than sensor.